test/app-tap/CMakeLists.txt

```diff
@@ -6,3 +6,103 @@ target_link_libraries(module_api msgpuck)
 set(CMAKE_C_FLAGS "-Wall -Wextra -std=c99")
 set(CMAKE_C_FLAGS_DEBUG "-g -O0 -Werror")
 set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2")
+
+make_lua_path(LUA_PATH
+  PATHS
+    ./?.lua
```

Comment:
It is better to use `${CMAKE_CURRENT_SOURCE_DIR}` instead of `.`.

Reply:
Fixed, thanks!

```diff
--- a/test/app-tap/CMakeLists.txt
+++ b/test/app-tap/CMakeLists.txt
@@ -9,7 +9,7 @@ set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2")
 
 make_lua_path(LUA_PATH
   PATHS
-    ./?.lua
+    ${CMAKE_CURRENT_SOURCE_DIR}/?.lua
     ${CMAKE_CURRENT_SOURCE_DIR}/lua/?.lua
     # Needed for test-run/test_run.lua.
     ${PROJECT_SOURCE_DIR}/test-run/?.lua
```